storage-proofs-core/src/api_version.rs

```diff
@@ -114,6 +114,31 @@ impl ApiFeature {
     }
 }
 
+/// Display is also used for command-line argument tooling.
+impl Display for ApiFeature {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let api_feature_str = match self {
+            Self::SyntheticPoRep => "synthetic-porep",
+            Self::NonInteractivePoRep => "non-interactive-porep",
+        };
+        write!(f, "{}", api_feature_str)
+    }
+}
+
+impl FromStr for ApiFeature {
+    type Err = Error;
+    fn from_str(api_feature_str: &str) -> Result<Self> {
+        match api_feature_str {
+            "synthetic-porep" => Ok(ApiFeature::SyntheticPoRep),
```

Comment:
Maybe also add short forms for convenience? e.g. "synporep, niporep"

Reply:
Those are used for benchy. I'd like to keep them close to the actual enums for consistency.